src/globus_sdk/_testing/data/compute/v2/get_result_amqp_url.py

```diff
@@ -0,0 +1,15 @@
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+DEFAULT_RESPONSE_DOC = {
+    "queue_prefix": "some_prefix",
+    "connection_url": "amqps://user:password@amq.fqdn",
+}
```

Comment:
The `p` was too much for this line?  :wink:

Reply:
Hah, nice catch!